src/storage/mvcc/mod.rs

```diff
@@ -383,7 +384,7 @@ pub mod tests {
         let cm = ConcurrencyManager::new(start_ts);
```

Comment:
I don't know where should I put these test util functions.
There're several options I have considered:
- In `actions::tests`
- In `actions::cleanup::tests` (since it is a "variant" of `must_cleanup`)

Reply:
Uses of test utils spread wider than the mod where it defines. Maybe I will put all of them in `action::tests`. I don't have a strong preference, though.